### ISSUE TITLE
initialClass/initialclass capitalization mismatch between documentation and `hyprctl clients` output versus hyprland.conf parsing

### ISSUE THREAD

**Issue by LJNielsenDk** [hyprwm/Hyprland#4965]
### Hyprland Version

Hyprland, built from branch HEAD at commit 84ab8d11e8951a6551d1e1bf87796a8589da6d47  (props: bump ver to 0.35.0).
Date: Mon Feb 5 01:59:02 2024
Tag: v0.35.0

flags: (if any)

### Bug or Regression?

Bug

### Description

The [window rule documentation](https://wiki.hyprland.org/Configuring/Window-Rules/) and the output of `hyprctl clients` has the spelling as initialClass but the rule doesn't work if the c is capitalized in the config file.

`windowrulev2 = float,initialclass:(KiCad)` works but `windowrulev2 = float,initialClass:(KiCad)` does not appear to have an effect. Neither capitalization result in a red error message is shown for some other config issues.

Same thing with both windowrule and windowrulev2.

### How to reproduce

See description.

**Comment by vaxerski** [comment]
yeah they are not meant to be the same, though I believe a better solution would be to allow any capitalization in wrv2

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
